specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1910,6 +2067,255 @@
+      "properties": {
+        "forecasts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AirQuality"
+          },
+          "description": "A list of all hourly air quality forecasts for the queried location."
+        }
+      }
+    },
+    "DailyAirQualityForecastResponse": {
+      "type": "object",
+      "description": "This object is returned from a successful Get Daily Air Quality Forecast call.",
+      "properties": {
+        "forecasts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DailyAirQuality"
+          },
+          "description": "A list of all daily air quality forecasts for the queried location."
+        }
+      }
+    },
+    "AirQuality": {
+      "type": "object",
+      "description": "Information about the air quality in a specific location at a specific time.",
+      "properties": {
+        "dateTime": {
+          "type": "string",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
+        },
+        "index": {
+          "type": "number",
+          "format": "float",
+          "description": "Air quality rating on a scale set by local regulating bodies. Scales can vary widely based on location. See [Wikipedia](https://en.wikipedia.org/wiki/Air_quality_index) for more information."
+        },
+        "globalIndex": {
+          "type": "number",
+          "format": "float",
+          "description": "Internationally normalized air quality rating on a scale from 0 to 300 and up, with higher numbers representing worse air quality."
+        },
+        "dominantPollutant": {
+          "type": "string",
+          "description": "The pollutant with the highest concentration."
+        },
+        "category": {
+          "type": "string",
+          "description": "One-word description of the air quality in the requested language. For example, \"Excellent\"."
+        },
+        "categoryColor": {
+          "type": "string",
+          "description": "A unique color corresponding to the category of this air quality result."
+        },
+        "description": {
+          "type": "string",
+          "description": "A textual explanation of this air quality result in the requested language."
+        },
+        "pollutants": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Pollutant"
+          },
+          "description": "Information about individual pollutants."
+        }
+      }
+    },
+    "DailyAirQuality": {
+      "type": "object",
+      "description": "Information about the air quality in a specific location at a specific time.",
+      "properties": {
+        "dateTime": {
+          "type": "string",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
```

Comment:
`"format": "date-time"` will be a good addition here as the proceeding codegen will result in date types rather then raw strings:  https://github.com/ambientlight/azure-sdk-for-go/blob/825aa43364c936dfde70094d465ab8e7e07eef10/services/preview/maps/1.0/weather/models.go#L313

Reply:
Updated